bindings/nodejs/lib/MessageHandler.ts

```diff
@@ -5,7 +5,6 @@ import {
     sendMessageAsync,
     messageHandlerNew,
     listen,
-    clearListeners,
```

Comment:
I'm a bit confused, what does it point to? And why is the thing it points to not removed?

Reply:
as far as i know, there was none ._.

Reply:
What do you mean? If we can import it, it means it exists?

Reply:
It was imported from the binary for nodejs, there we removed it already earlier, so all good removing it now also from js side